### DISCUSSION TITLE
The last edit time recorded in the blog project post of the associated github deployment is incorrect

### DISCUSSION THREAD

**Discussion by SherryMW** [vercel/community#1238] [Help]
### Summary

I found in the past two days that the last edit time recorded in GitHub for my latest updated blog posts is correct, but the last edit time recorded for all the old blog posts is wrong, I ran it locally with no problem, and found this problem after I submitted it to GitHub and published it with vercel

**Comment by SherryMW** [comment]
![image](https://user-images.githubusercontent.com/118236482/211459243-f290fddb-0da0-4312-a109-3502c79a1d23.png)
I can see that the last edit time recorded by GitHub is accurate when running the project locally, but after deploying to production via vercel, the last edit time recorded is incorrect, and I am troubled that every time I update a new article, the last edit time of all the old articles is updated once and incorrectly

**Comment by amyegan** [comment]
Hi @SherryMW! Do you have a public repo or [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that you can share? This sounds like it could be a problem with a serverless function, but I don't know enough about your project to pinpoint the source of the error.

**Reply by SherryMW** [reply]
HI amyegan！Thanks for your reply.  
my blog public repository address is [https://github.com/SherryMW/Blog](https://github.com/SherryMW/Blog)  
The blog is built on vuepress framework and the deployment configuration on vercel is  
![image](https://user-images.githubusercontent.com/118236482/211698117-87970cd1-31e4-4e96-aae9-851c0e9e306a.png)

**Reply by SherryMW** [reply]
Because the problem has never occurred before, it has only appeared in the last few days.  
I turned on the hotReload configuration item in my local Blog/docs/src/.vuepress/theme.ts configuration file, which enables Git-based features, including contributors, automatic creation dates, and last update times.  So I see the corresponding github commit time for each article locally, and the last edit time for all articles in the locally running project is correct.  
I guess vercel is also fetching the latest github commit.However, after publishing via vercel, it was found that the last edit time of all the old articles that had not been updated had changed incorrectly, except for the latest article submitted for update which showed the correct github last edit time
I hope my description can make you understand my confusion
